web/pgadmin/browser/static/js/keyboard.js

```diff
@@ -193,8 +193,10 @@ _.extend(pgBrowser.keyboardNavigation, {
     else if (combo.key === shortcut_obj.close_tab_panel) {
       const panelId = dockLayoutTabs[activeTabIdx].id?.slice(14);
       if (panelId) {
-        pgAdmin.Browser.docker.close(panelId);
-        activeTabIdx = activeTabIdx === dockLayoutTabs.length - 1 || activeTabIdx === 0 ? 1 : activeTabIdx + 1;
+        const warningPrompt = pgAdmin.Browser.docker.close(panelId);
```

Comment:
This is not the right way to do this, this is a hack. There other many tools in pgAdmin which closes with a warning. Putting `warningPrompt` variable all over the places is preposterous.